packages/ui/components/form-core/test-suites/FormRegistrationMixins.suite.js

```diff
@@ -256,6 +257,23 @@ export const runRegistrationSuite = customConfig => {
       ]);
     });
 
+    describe('FormRegisteringMixin', () => {
+      it('dispatches the form-element-register event with correct details upon connectedCallback', async () => {
+        const el = /** @type {RegistrarClass} */ (
+          await fixture(html`
```

Comment:
It would be best to prove the use case here: add childTag as part of the shadow dom of parent (by creating a separate class).

And then it would also be good to test in a separate fixture that the spy will not be called without `allowCrossRootRegistration`

Reply:
Added a case for a spy not called if the flag is not set